src/middleware/authUser.js

```diff
@@ -0,0 +1,8 @@
+const { verifyRole, addUserIdToSession } = require('./authHelpers.js')
```

Comment:
Yeah I thought after taking it out it then seemed a bit pointless/unreadable, but then it was mostly to test, I'll do it this way instead